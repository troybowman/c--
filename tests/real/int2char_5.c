/* assignment: char --> int conversion: locals-->globals */

int x;

extern void printf(char format[], ...);

void main(void)
{
  char y;

  y = 'Q';
  x = y;

  printf("x: %d %c\n", x, x);
  printf("y: %c %d\n", y, y);
}
