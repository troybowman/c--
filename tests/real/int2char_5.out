x: 81 Q
y: Q 81
